engine/gamesys/src/gamesys/components/comp_gui.cpp

```diff
@@ -365,6 +365,7 @@ namespace dmGameSystem
         tex_params.m_DataSize = sizeof(white_texture);
         tex_params.m_Width = 2;
         tex_params.m_Height = 2;
+        tex_params.m_Depth = 1;
```

Comment:
From the main repo, making sure we make correct calculations, when passing to graphics backends.